docs/appendices/release-notes/unreleased.rst

```diff
@@ -205,3 +205,9 @@ Fixes
   consume invalid table names provided in a double-quoted string format
   containing ``.`` such as ``"table.t"`` by mis-interpreting it as
   ``"table"."t"``, which is a two double-quoted strings joined by a ``.``.
+
+- Fixed an issue that caused failure of the statements containing comparison of
+  :ref:`bit <data-type-bit>` strings with different length. An example::
```

Comment:
initially was not going a change entry since originally issue was reported on unreleased bit operators

SELECT * FROM tbl WHERE b & B'100' =  B'100'

But since comparing BIT strings even with different length is a valid scenario I  added a change entry